### ISSUE TITLE
New request: Atheist Republic

### ISSUE THREAD

**Issue by SecularSekai** [openzim/zim-requests#761]
*Please use the following format for a ZIM creation request (and delete unnecessary information)*

- Website URL: https://www.atheistrepublic.com/
- License: Copyright owner provided permission for archive request (susanna@atheistrepublic.com)
- Desired ZIM Title: Atheist Republic
- Desired ZIM Description: We are not just atheists, we are atheists who care.
- Desired ZIM Icon –png (URL or attach one): https://www.atheistrepublic.com/sites/default/files/logo-3rev-fbfix1.png
- Language (ISO 639-3): Eng
- Is this a MediaWiki?: no

**Comment by Popolechien** [comment]
@SecularSekai Hey can you please drop a line at hello@kiwix.org for the permission?

**Comment by RavanJAltaie** [comment]
@Popolechien please let up know here once permission is sent so we can start creating the recipe. @SecularSekai

**Comment by SecularSekai** [comment]
@Popolechien @RavanJAltaie Sorry for the wait! I notified the copyright holder who has informed me that she sent an email providing permission to hello@kiwix.org
If you need any additional information, please let me know. Thanks, guys!

**Comment by Popolechien** [comment]
@RavanJAltaie We're good to go - fingers crossed.

**Comment by RavanJAltaie** [comment]
https://farm.openzim.org/recipes/atheistrepublic_en_all
Recipe created, will update the library link here once ready

**Comment by SecularSekai** [comment]
@RavanJAltaie Fantastic! How can I access it and view the ZIM locally on Kiwix Desktop?

**Comment by SecularSekai** [comment]
@RavanJAltaie @Popolechien Hi, guys! It looks like the recipe failed when I check the link. What would be the next step to troubleshoot it?

**Comment by RavanJAltaie** [comment]
@benoit74 the recipe is failing with error:
`File "/usr/bin/zimit", line 566, in <module>
    zimit()
  File "/usr/bin/zimit", line 437, in zimit
    raise subprocess.CalledProcessError(crawl.returncode, cmd_args)
subprocess.CalledProcessError: Command '['crawl', '--failOnFailedSeed', '--waitUntil', 'load', '--title', 'Atheist Republic', '--description', 'We are not just atheists, we are atheists who care.', '--depth', '-1', '--timeout', '90', '--scopeType', 'domain', '--lang', 'eng', '--behaviors', 'autoplay,autofetch,siteSpecific', '--behaviorTimeout', '90', '--diskUtilization', '90', '--url', 'https://www.atheistrepublic.com/', '--userAgent', 'Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/605.1.15 (KHTML, like Gecko) Version/17.0 Safari/605.1.15 +Zimit contact+zimfarm@kiwix.org', '--cwd', '/output/.tmpmypky3kv', '--statsFilename', '/output/crawl.json']' returned non-zero exit status 9.`
Any ideas?

**Comment by benoit74** [comment]
It looks after some time the scraper has been banned due to too many requests.

However, digging a bit into the logs it is clear that we spend a lot of time (more than 90% I would say) trying to archive the store items which is probably not intentional. I don't think this is needed to be present in the ZIM.

I suggest that we exclude all `https://www.atheistrepublic.com/store` URLs.

I also noticed there is a forum which is hosted both on https://www.atheistrepublic.com/forums/ (old forum if I get it right) and https://forum.atheistrepublic.com/ (new forum). New forum seems to be based on Discourse platform, which will probably be difficult to scrape (we might be blocked quite soon).

@SecularSekai do you want the forums to be archived in the ZIM as well or is it not needed or not mandatory?

**Comment by RavanJAltaie** [comment]
@benoit74 what do you think we should do?
Shall we tag this as upstream or reject the issue?

**Comment by SecularSekai** [comment]
@benoit74 Hi! Sorry I didn't see your comment back in February until now. The forums are less important, so they can be omitted if need be for the ZIM

We really appreciate all the help with this!

**Comment by RavanJAltaie** [comment]
@benoit74 shall we try create it without the forums?

**Comment by benoit74** [comment]
Next steps are:
- Confirm that we can get rid of store items
- If we all agree that these are not needed, get rid of these store items 
- Run again the recipe and see if forums are working ok or not.
- If not, then get rid of forum items as well and run again the recipe

Do we all agree that we do not want store content inside the ZIM?

**Comment by RavanJAltaie** [comment]
@benoit74 I agree on all points.
@SecularSekai do you agree?

**Comment by SecularSekai** [comment]
@RavanJAltaie @benoit74 I agree and appreciate the thorough thought on this. Let's move forward with that strategy and see where we get.
Loss of the forums is not a major problem.

**Comment by benoit74** [comment]
@RavanJAltaie do you need help to remove the store items?

**Comment by RavanJAltaie** [comment]
@benoit74 yes please, you can let me know how to be done and I'll do it.

**Comment by benoit74** [comment]
I've updated the recipe exclude criteria:

![image](https://github.com/openzim/zim-requests/assets/7102089/f11194c3-180d-43e7-aa45-835830b0821e)

This will exclude store items for now. Forum should be archived since it is on the same domain and scopeType is set to domain. Let's see if it achieves to properly archive the forum as well. I've requested the recipe.

**Comment by Popolechien** [comment]
Updated recipe at https://farm.openzim.org/recipes/atheistrepublic.com_en_all/edit and created CMS entry at https://cms.openzim.org/title/atheistrepublic.com_en_all

Awaiting review

**Comment by benoit74** [comment]
Recipe enabled and requested, let's see

**Comment by benoit74** [comment]
Website is down ATM, I cancelled the task

**Comment by Popolechien** [comment]
Website is back up but the task failed anyway - not too sure about root cause tbh

**Comment by benoit74** [comment]
Crawler crashed, might be solved by https://github.com/openzim/zimit/issues/436 once implemented
